ruby/rbenv/Portfile

```diff
@@ -4,9 +4,10 @@ PortSystem          1.0
 PortGroup           github 1.0
 
 github.setup        rbenv rbenv 1.2.0 v
+revision            1
 license             MIT
 categories          ruby
-platforms           any
+platforms           darwin
```

Comment:
Is there a reason for this change? Since rbenv is just a bash script, I believe it should run on other platforms as well.

Reply:
The change here was related to `port lint`. I have started following more MacPorts messages, and it seems that `port lint` has not been updated to recognize `any` as a valid value, although it is the correct choice. I will revert it (I will resolve after I have made the change).